src/tests/ftest/daos_test/dfuse.py

```diff
@@ -93,6 +93,8 @@ def run_test(self, il_lib=None):
             daos_test_env['D_LOG_MASK'] = 'INFO,IL=DEBUG'
 
         command = [self.daos_test, '--test-dir', mount_dir, '--io', '--stream']
+        if use_dfuse:
+            command.append('--lowfd')
```

Comment:
Is there a reason for only enabling this when dfuse is enabled?  The purpose of this test is to ensure dfuse or interception behave the same as natively, does this test fail when run on a native filesystem?

Reply:
This test fails under a native filesystem. consume_low_fd() and daos_init() are called on the first time accessing file/dir on DFS. They are not triggered when only native filesystem is used.